source/js/buyers-guide/template-js-handler/product-page-comment-handler.js

```diff
@@ -4,34 +4,38 @@
  * we have this file which will push a datalayer event whenever a user submits a comment.
  */
 export default () => {
+  // Commento's parent div, in which it loads all of its content through JS.
   const commentoContainer = document.querySelector(
     "#view-product-page #commento"
   );
-  const submitButton = document.querySelector("#commento-submit-button-root");
 
-  if (commentoContainer && submitButton) {
-    submitButton.addEventListener(`click`, () => {
-      commentSubmitted();
-    });
+  if (!commentoContainer) {
+    return;
   }
 
-  function commentSubmitted() {
-    // Checking if the user is logged in and if the text box has a value. 
-    // If so, push the DataLayer event.
-    let loggedInContainer = document.querySelector(
-      "#commento-logged-container"
-    );
-    let textBoxValue = document.querySelector("#commento-textarea-root").value;
-
-    if (loggedInContainer && textBoxValue) {
-      window.dataLayer = window.dataLayer || [];
+  const commentListUpdateHandler = (mutations) => {
+    mutations.forEach(function (mutation) {
+      // New comments are appended in the form of: <div><div class='commento-card'/></div>
+      // If comment tracking ever breaks, this logic be a good place to check first.
+      if (
+        mutation.addedNodes[0].firstChild.classList.contains("commento-card")
```

Comment:
This is what a "new comment" event currently looks like:
- A mutation with 1 added node, which is a div.
<img width="1192" alt="Screen Shot 2022-10-03 at 12 19 56 PM" src="https://user-images.githubusercontent.com/18314510/193662403-8262b584-e527-43eb-bdff-25128e2a6341.png">

- This new div has one child, which is the new comment div with the classname of 'commento-card'

<img width="1192" alt="Screen Shot 2022-10-03 at 12 20 57 PM" src="https://user-images.githubusercontent.com/18314510/193662492-17fc51c7-3354-463e-aabe-a88ca15aae05.png">


This If statement is saying "if the first child of the added node has the class 'commento-card', its safe to say a comment has been submitted and we can push the datalayer event.